composer.json

```diff
@@ -26,6 +26,6 @@
     "scripts": {
         "style:check": "docker-compose -f docker-compose.cli.yml run --rm php-cs-fixer fix --verbose --show-progress=estimating --dry-run",
         "style:fix": "docker-compose -f docker-compose.cli.yml run --rm php-cs-fixer fix --verbose --show-progress=estimating",
```

Comment:
This will run inside the container, so I'd remove the from here and add the instructions to the readme (we need to create one):

```bash
✗  docker compose -f docker-compose.cli.yml run --rm php-cs-fixer fix --verbose --dry-run
PHP CS Fixer 3.1.0 River by Fabien Potencier and Dariusz Ruminski (cf4cedb)
Runtime: PHP 7.1.33
Loaded config default from "/data/.php-cs-fixer.php".
........                                                                                                                                                                                  8 / 8 (100%)
Legend: ?-unknown, I-invalid file syntax (file ignored), S-skipped (cached or empty file), .-no changes, F-fixed, E-error

Checked all files in 0.090 seconds, 12.000 MB memory used
```

So the readme should have a section for installing dependencies:

```bash

docker compose run --no-deps --rm php-cli composer install
```

and another to execute lint:

```bash
docker compose -f docker-compose.cli.yml run --rm php-cs-fixer fix --verbose --dry-run
```

Reply:
[e22ce50](https://github.com/TiendaNube/image-compression-lib/pull/13/commits/e22ce504d1c3f2f4337e626e6146b2ce4fc23986)